benches/README.md

```diff
@@ -0,0 +1,28 @@
+# jsonrpsee benchmarks
+
+This crate contains benchmarks mainly to test the server implementations of some common scenarios such as concurrent connections.
+Further, running these will open lots of sockets and file descriptors, it doesn't work well on macOS for instance.
```

Comment:
Is it worth suggesting some commands that can be run to help with this? (eg https://github.com/paritytech/substrate-telemetry/blob/d6ed878cb41a97f7b7b546e996f46ac5dd6215aa/backend/telemetry_core/tests/soak_tests.rs#L22 .. not sure how many of those are actually useful but I just paste them all each time!)